api.ts

```diff
@@ -2067,8 +2137,23 @@ export function makeDynamicSyncTable<
     formula,
     connectionRequirement,
     dynamicOptions: {getSchema, entityName, defaultAddDynamicColumns},
-    propertyAutocomplete,
```

Comment:
this is the unused autocomplete field used by 2-way right?

Reply:
Yeah, that field was added by me for the earlier implementation where you'd always define `propertyAutocomplete` on a sync table itself. That kind of still exists here but it's only intended for dynamic sync tables and got renamed to just `autocomplete`